cmd/sourced/release/release_test.go

```diff
@@ -0,0 +1,88 @@
+package release
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/src-d/sourced-ce/cmd/sourced/dir"
+	"github.com/stretchr/testify/assert"
+)
+
+type testCase struct {
+	responseTag string // tag returned by github
+	current     string
+	update      bool
+	latest      string
+}
+
+func TestFindUpdatesSuccess(t *testing.T) {
+	os.RemoveAll(filepath.Join(dir.TmpPath(), "httpcache"))
+
+	assert := assert.New(t)
```

Comment:
I believe `assert` needs to be initialized with the `t` inside each `t.Run`, otherwise it will not pick up it's a sub test
```go
func TestFindUpdatesSuccess(t *testing.T) {
	os.RemoveAll(filepath.Join(dir.TmpPath(), "httpcache"))

	cases := []testCase{
		{
			responseTag: "v0.14.0",
			current:     "v0.14.0",
			update:      false,
			latest:      "0.14.0",
		},
		{
			responseTag: "v0.11.0",
			current:     "v0.14.0",
			update:      false,
			latest:      "0.11.0",
		},
		{
			responseTag: "v0.14.0",
			current:     "v0.13.0",
			update:      true,
			latest:      "0.14.0",
		},
		{
			responseTag: "v0.14.0",
			current:     "v0.13.1",
			update:      true,
			latest:      "0.14.0",
		},
	}

	for _, c := range cases {
		name := fmt.Sprintf("%s_to_%s", c.current, c.responseTag)
		t.Run(name, func(t *testing.T) {
			assert := assert.New(t)

			restore := mockGithub(c.responseTag)
			defer restore()

			update, latest, err := FindUpdates(c.current)
			assert.Nil(err)
			assert.Equal(c.update, update)
			assert.Equal(c.latest, latest)
		})
	}
}
```

Reply:
fixed